vkcs/resource_vkcs_db_cluster.go

```diff
@@ -176,16 +176,24 @@ func resourceDatabaseCluster() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"uuid": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+							Description: "The id of the network. Changing this creates a new cluster." +
+								"**Note** Using this field as optional is deprecated. It will be marked as required in a future major release.",
```

Comment:
dinap whants to make it optional in fact. So we should say that it is really required at the moment despite it is described as optional. And in the future we remove the note

Reply:
Updated:
https://github.com/vk-cs/terraform-provider-vkcs/blob/a9ed34cda116797e1c99ff3de54c31bd0d7a2138/vkcs/resource_vkcs_db_cluster.go#L178-L184